query_table_create.go

```diff
@@ -12,6 +12,10 @@ import (
 	"github.com/uptrace/bun/schema"
 )
 
+type IdentityAppender interface {
+	AppendIdentity(b []byte) []byte
+}
```

Comment:
Seems to be unused. Let's remove.

Reply:
Yeah, sorry, forgot to remove that. Done.